_docs/geyser/geyser-api.md

```diff
@@ -0,0 +1,102 @@
+- A mod for Fabric or Forge
+- A Geyser Extension
+
+### Accessing the Geyser API
+See [here](/geyser/getting-started-with-api) for how to include the Geyser API dependency in your project.
+
+### Documentation
+
+The Geyser API offers events to subscribe to, or information on whether a player is joining through Geyser, and gives you the ability to enhance what is possible with Geyser (i.e. register custom items).
+(soon, blocks and entities too).
+It can be used easily in Geyser Extensions, see [here](/other/extensions) for details on those.
+
+**Quick overview:** <br>
+<div class="alert alert-info" role="alert">
+	Note: To see full, detailed documentation, see the linked sources with javadocs.
+</div>
+
+#### [GeyserAPI](https://github.com/GeyserMC/Geyser/blob/master/api/src/main/java/org/geysermc/geyser/api/GeyserApi.java):
+The GeyserApi interface serves as a central access point to various functionalities provided by the Geyser API, providing methods to e.g. interact with player connections.
+It extends the [BaseAPI](https://github.com/GeyserMC/api/blob/master/base/src/main/java/org/geysermc/api/GeyserApiBase.java) interface, which provides basic information about individual players.
+
+The class GeyserApi is the base class of the API and you need to use it to access any part of the API.<br>
+To access it, you simply type:
+```java
+GeyserApi.api();
+```
+
+After you got the instance, you have access to all the methods.<br>
+Use the documentation in the API module to see (and get info about) every single method available.  
+Most [API methods have a simple explanation](https://github.com/GeyserMC/Geyser/blob/master/api/src/main/java/org/geysermc/geyser/api/GeyserApi.java).
+Since the Geyser API extends the Base API that is shared across Floodgate and Geyser, you can also use the [methods from the Base API](https://github.com/GeyserMC/api/blob/master/base/src/main/java/org/geysermc/api/GeyserApiBase.java).
+
+
+#### We'll highlight a few to get you started quickly:
+`GeyserApi#isBedrockPlayer(UUID)`<br>
+Used to check if the given UUID of an **online** player is a Bedrock player.
+
+`GeyserApi#connectionByUuid(UUID)`<br>
+Used to get the [Connection](https://github.com/GeyserMC/api/blob/master/base/src/main/java/org/geysermc/api/connection/Connection.java) of an **online** player.<br>
+This method will return null if the player is not a Bedrock player.
+
+<div class="alert alert-info" role="alert">
+	You don't need to wait until the Bedrock player is online to use the getPlayer and isBedrockPlayer methods.<br>
+    You can even use them in the pre-login events.
+</div>
+
+`GeyserApi#sendForm(UUID, Form(Builder))`<br>
+Used to send a form to the Bedrock player with the given UUID.<br>
+Click [here](/floodgate/forms/) to get more information about Forms.
+
+`GeyserApi#onlineConnectionsCount()`<br>
+Used to get the amount of online Bedrock players.
+
+### Short Overview of the API
```

Comment:
More a general question; should we already prepare for the Floodgate merge?
Because a lot of the stuff mentioned below aren't part of the base api and I think that that difference isn't mentioned enough. And this page is mainly focused on the geyser variant of the api.

Reply:
The geyser-api.md page should only show the geyser api imo; the floodgate-api page is separate.
With the merge, it would probably make sense to replace the floodgate-api page with a base-api page, and tell people that the Geyser API extends the base one